letsencrypt/client/dns_authenticator.py

```diff
@@ -0,0 +1,254 @@
+"""DNS Authenticator"""
+import dns.query
+import dns.tsigkeyring
+import dns.update
+from dns.query import UnexpectedSource, BadResponse
+from dns.resolver import NoAnswer
```

Comment:
nit: could you please use module imports (`import dns.resolver`) and `dns.query.UnexpectedSource`, `dns.query.BadResponse`, `dns.resolver.NoAnswer` throughout the code below?